src/xpcc/architecture/platform/driver/fsmc/stm32/fsmc.cpp.in

```diff
@@ -9,15 +9,15 @@
 
 #include "fsmc.hpp"
```

Comment:
Where are the actual device files that you re-generated?

Somehow the commit message does not make sense to me in this context.
Isn't it rather "enable fsmc on STM32F7 device"?